src/ReactiveUI/ReactiveUI_UWP.csproj

```diff
@@ -72,6 +72,7 @@
     <Compile Include="MessageBus.cs" />
     <Compile Include="SuspensionHost.cs" />
     <Compile Include="ObservableAsPropertyHelper.cs" />
+    <Compile Include="Observables.cs" />
```

Comment:
It's doable with shared projects now...probably not worth the faff of doing it twice though!